test/gerrit_tests/test_gerrit.py

```diff
@@ -388,6 +397,145 @@ def test_request_reviews_no_repos(self,
         mock_format_response.assert_not_called()
         self.assertEqual(None, response)
 
+    @patch(PATH + 'GerritService.format_response')
```

Comment:
where does PATH come from?

Reply:
It's a global in test_gerrit.py. Existing tests were already using it.